### PR TITLE
fix: fixed embed where floating button was hovering over modal on mobile

### LINKED ISSUE #8851: [CAL-1655] Embed on mobile button stays above booking modal
So I can’t actually click to confirm the booking. This floating button should disappear while modal is open. 

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-1655](https://linear.app/calcom/issue/CAL-1655/embed-on-mobile-button-stays-above-booking-modal)</sub>

### PR DESCRIPTION
**Pull request by doug-andrade**
## What does this PR do?

Keeps the embedded floating under the modal while the modal is open.

Fixes #8851 

https://www.loom.com/share/c3960f30740a4ff8babcb6ef3dce8215

**Environment**: Staging(main branch)

## Type of change

- Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

**Review by hariombalhara — CHANGES_REQUESTED** [review]
Left a comment

File: packages/embeds/embed-core/src/FloatingButton/FloatingButtonHtml.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ const getHtml = ({
 }) => {
   // IT IS A REQUIREMENT THAT ALL POSSIBLE CLASSES ARE HERE OTHERWISE TAILWIND WONT GENERATE THE CSS FOR CONDITIONAL CLASSES
   // To not let all these classes apply and visible, keep it hidden initially
-  return `<button class="z-[10000000000] hidden fixed md:bottom-6 bottom-4 md:right-10 right-4 md:left-10 left-4 ${buttonClasses.join(
+  return `<button class="z-[1000000] hidden fixed md:bottom-6 bottom-4 md:right-10 right-4 md:left-10 left-4 ${buttonClasses.join(
```
[the remark this answers is not in the document]
Comment (PeerRich) [reply packages/embeds/embed-core/src/FloatingButton/FloatingButtonHtml.ts:14]: @doug-andrade can you take a look?

[the remark this answers is not in the document]
Comment (PeerRich) [reply packages/embeds/embed-core/src/FloatingButton/FloatingButtonHtml.ts:14]: I think hiding when open is better

[the remark this answers is not in the document]
Comment (doug-andrade) [reply packages/embeds/embed-core/src/FloatingButton/FloatingButtonHtml.ts:14]: @PeerRich 100% Honesty, I would need more time to learn the codebase/the github workflow before I'd feel confident taking that on. I'm going to close this PR for now and will reopen if I can figure that out.

**Pull request closed without merging**